exchange-v2/contracts/lib/LibSignature.md

```diff
@@ -0,0 +1,16 @@
+#### Features
+
+The library recovers address of the signer from a signature.
+
+## recover(bytes32 hash, bytes memory signature)
+- recovers r,s,v values from signature
+- calls recover(bytes32 hash, uint8 v, bytes32 r, bytes32 s)
+## recover(bytes32 hash, uint8 v, bytes32 r, bytes32 s)
+- checks values of s and v for correctness
+- check for v > 30 case
+    - if v > 30, then the signature came from wallet that doesn't support EIP-712 type of signatures
+    - in that case, we eneed to adjust hash of the message with "\x19Ethereum Signed Message:\n32"
```

Comment:
typo. pls check, may be there are more typos

Reply:
fixed